mars/services/scheduling/supervisor/autoscale.py

```diff
@@ -95,19 +98,33 @@ async def request_worker(
                 time.time() - start_time,
             )
 
-    async def release_workers(self, addresses: List[str]):
+    async def disable_autoscale_in(self):
+        self._autoscale_in_disable_counter += 1
+        if self._autoscale_in_disable_event.is_set() and self._enabled:
```

Comment:
Seems like `self._autoscale_in_disable_event` is not necessary, can we remove it?

Reply:
Good catch, I'll remove it.